py/torch_tensorrt/_compile.py

```diff
@@ -93,13 +100,14 @@ def _get_target_ir(module_type: _ModuleType, ir: str) -> _IRType:
     else:
         if ir == "default":
             # Options are listed in order of preference
-            if module_is_fxable:
+            if DYNAMO_ENABLED and module_is_fxable:
```

Comment:
Update default path to be TorchScript if the install is with a legacy Torch version